gulpfile.js

```diff
@@ -280,11 +296,106 @@ function debug(done) {
     var platforms = getPlatforms();
     removeItem(platforms, 'chromeos');
 
-    buildNWApps(platforms, 'sdk', DEBUG_DIR, done);
+    buildNWAppsWrapper(platforms, 'sdk', DEBUG_DIR, done);
 }
 
-function buildNWApps(platforms, flavor, dir, done) {
+function injectARMCache(flavor, callback) {
+    var flavorPostfix = `-${flavor}`;
+    var flavorDownloadPostfix = flavor !== 'normal' ? `-${flavor}` : '';
+    clean_cache().then(function() {
+        if (!fs.existsSync('./cache')) fs.mkdirSync('./cache');
```

Comment:
Please use braces after all conditional / loop statements - this makes code a lot more maintainable.